src/SFA.DAS.AdminService.Web/Views/CertificateAmend/Check.cshtml

```diff
@@ -189,75 +189,78 @@ else
+                            <a class="govuk-link" asp-controller="CertificateRecipient" asp-action="Recipient" asp-route-fromapproval="@Model.FromApproval" asp-route-certificateId="@Model.Id" asp-route-stdcode=@Model.StandardCode asp-route-uln=@Model.Uln asp-route-searchstring=@Model.SearchString>
+                                Change<span class="govuk-visually-hidden"> recipient name</span>
+                            </a>
+                        </dd>
+                    </div>
+                    <div class="govuk-summary-list__row">
+                        <dt class="govuk-summary-list__key">
+                            Department
+                        </dt>
+                        <dd class="govuk-summary-list__value">
+                            @Model.Dept
+                        </dd>
+                        <dd class="govuk-summary-list__actions">
+                            <a class="govuk-link" asp-controller="CertificateRecipient" asp-action="Recipient" asp-route-fromapproval="@Model.FromApproval" asp-route-certificateId="@Model.Id" asp-route-stdcode="@Model.StandardCode" asp-route-uln="@Model.Uln" asp-route-searchstring="@Model.SearchString">
+                                Change<span class="govuk-visually-hidden"> department</span>
+                            </a>
+                        </dd>
+                    </div>
+                    <div class="govuk-summary-list__row">
+                        <dt class="govuk-summary-list__key">
+                            Organisation
+                        </dt>
+                        <dd class="govuk-summary-list__value">
+                            @Model.Employer
+                        </dd>
+                        <dd class="govuk-summary-list__actions">
+                            <a class="govuk-link" asp-controller="CertificateAddress" asp-action="Address" asp-route-fromapproval="@Model.FromApproval" asp-route-certificateId=@Model.Id asp-route-stdcode="@Model.StandardCode" asp-route-uln="@Model.Uln" asp-route-searchstring="@Model.SearchString">
+                                Change<span class="govuk-visually-hidden"> organisation</span>
+                            </a>
+                        </dd>
+                    </div>
+                    <div class="govuk-summary-list__row">
+                        <dt class="govuk-summary-list__key">
+                            Address
+                        </dt>
+                        <dd class="govuk-summary-list__value">
+                            @Model.AddressLine1<br />
+                            @if (@Model.AddressLine2 != null)
+                            {
+                                @Model.AddressLine2<br />
+                            }
+                            @if (@Model.AddressLine3 != null)
+                            {
+                                @Model.AddressLine3<br />
+                            }
```

Comment:
This is existing code but a better check would have been !string.IsNullOrWhitespace(....)